packages/docs/docs/docs/architecture.md

```diff
@@ -0,0 +1,97 @@
+---
+id: architecture
+title: On-chain Architecture
+---
+
+ZeppelinOS's features such as upgradeability, EVM package linkage, etc. can all be used independently as standalone components. However, they work best together when used within a full ZeppelinOS project. This can be done using the command:
+
+TODO: what is the command?
+```
+zos init --full
+```
+
+A ZeppelinOS project is managed by a set of on-chain contracts that can be found in [zos/packages/lib/contracts/application](https://github.com/zeppelinos/zos/tree/master/packages/lib/contracts/application). It is important to understand that these contracts are not strictly necessary to work with ZeppelinOS, however, they provide a lot of convenience and it is recommended to start new projects with them.
```

Comment:
I'd suggest to change references to `master` to references to the specific tag for the version (once the version is tagged, of course). If those contracts are moved later, we'll end up with dead links.

Reply:
@spalladino I like the suggestion, but will keep the current link until we have a tag.
@elopio I re-worded this part a bit, can you please review it again?